### PR TITLE
add DESCRIPTION, make renv install modelstats

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**
# Purpose of this PR
Add a DESCRIPTION file to document dependencies and other metadata in a standard way. Renv would not realize `modelstats` was a dependency (for rs2), so it is now listed explicitly as a dep in DESCRIPTION.

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [x] Bug fix 

## Checklist:

- [x] My code follows the coding etiquette
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,75 @@
+Package: remindmodel
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline DESCRIPTION:1]: REMIND is not a package.

Comment (pfuehrlich-pik) [reply DESCRIPTION:1]: That's true. I got the idea to create a DESCRIPTION anyways from [Hadley Wickham](https://r-pkgs.org/Metadata.html), all the way at the bottom he describes this off-label use of a DESCRIPTION. Tools like renv and devtools can work with this DESCRIPTION even if the whole thing is not a package. Apparently the way you set a project's name in a DESCRIPTION is via the `Package` field, in the `Description` field it is clearly state that this is not a package though.

Comment (giannou) [reply DESCRIPTION:1]: I'm fine with adding the DESCRIPTION file. Will it then complement the automatic installation of needed packages and dependencies by `renv`?

Comment (pfuehrlich-pik) [reply DESCRIPTION:1]: Yes, `renv::dependencies` will check the DESCRIPTION in addition to scanning all R and Rmd files to determine dependencies, and this is what's installed when running R with an empty renv.

Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply DESCRIPTION:1]: > I got the idea to create a DESCRIPTION anyways from [Hadley Wickham](https://r-pkgs.org/Metadata.html)

… from Hadley Wickham's book on writing R packages, which REMIND isn't.

`renc::dependencies()` also parses DESCRIPTION files without a `Package` field.

Comment (pfuehrlich-pik) [reply DESCRIPTION]: I removed the `Package: remindmodel` line

**Review by giannou — APPROVED** [review]

**Pull request merged**